content/en/serverless/azure_app_services/azure_app_services_linux.md

```diff
@@ -144,6 +146,8 @@ See [Organization Settings > API Keys][301] in Datadog.<br>
 Your [Datadog site][302]. Defaults to `datadoghq.com`.<br>
 Use the "Datadog Site" drop-down menu on this page's right navigation bar to select your site.<br>
 
+**Logging and Service Configuration**
```

Comment:
I'd would rename this section to "Unified Service Tagging" and keep `DD_SERVICE`, `DD_ENV`, and `DD_VERSION`. See https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging/

Then start a new "Logging Configuration" section below with `DD_SOURCE`.

Reply:
Yeah, good point, they're not directly related to logging.